app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ExposureNotificationRepository.kt

```diff
@@ -162,6 +182,21 @@ class ExposureNotificationRepositoryImpl(
         exposureNotificationClient.isEnabled
             .addOnSuccessListener { enabled: Boolean ->
                 frameworkEnabledState.onNext(enabled)
+                //call it here again as we want to keep the state most up to date
```

Comment:
If you are registering bluetoothReceiver here, remove it from start and stop methods. Use only one way for it.